internal/config/config.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package config
+
+import (
+	"github.com/edgexfoundry/edgex-cli/internal/service"
+)
+
+var Configuration HostConfiguration
+
+type HostConfiguration struct {
+	CoreServices        Services
+	DeviceServices      Services
+	ApplicationServices Services
+}
+
+type Services map[string]service.Service
+
+func GetCoreService(name string) service.Service {
+	return Configuration.CoreServices[name]
+}
+
+func GetDeviceService(name string) service.Service {
+	return Configuration.DeviceServices[name]
+}
+
+func GetAppService(name string) service.Service {
+	return Configuration.ApplicationServices[name]
+}
+
+func init() {
+	Configuration.CoreServices = Services{
+		"core-metadata": {
```

Comment:
```suggestion
		common.CoreMetaDataServiceKey : {
```

Use similar constants for the others